broker/pkg/platform/kube/crd/client_test.go

```diff
@@ -26,23 +24,11 @@ import (
 	"istio.io/istio/broker/pkg/model/config"
 	"istio.io/istio/broker/pkg/testing/mock"
 	"istio.io/istio/broker/pkg/testing/util"
+	"istio.io/istio/tests/k8s"
 )
 
 func kubeconfig(t *testing.T) string {
-	usr, err := user.Current()
-	if err != nil {
-		t.Fatal(err.Error())
-	}
-	f := usr.HomeDir + "/.kube/config"
-
-	// For Bazel sandbox, the config is mounted in a different location
-	// as specified in prow/broker-presubmit.sh
-	if _, err = os.Stat(f); err != nil {
-		wd, _ := os.Getwd()
-		f = wd + "/../config"
-	}
-
-	return f
+	return k8s.Kubeconfig("/../../../../../pilot/platform/kube/config")
```

Comment:
Where's the file being symlinked?

Reply:
In the old bazel - I'm not changing anything, I assume scripts will do it.

For go/make: it is no longer used. Once bazel is out I'll also remove this line, and only use 
KUBECONFIG and the localhost/standalone apiserver conf for unit tests, and whatever the new
testing framework uses for integration ( most tests can be run in multiple environments ).